src/Cake.Jira/Cake.Jira.csproj

```diff
@@ -26,8 +26,6 @@
   <ItemGroup>
     <PackageReference Include="Atlassian.SDK" Version="10.4.0" />
     <PackageReference Include="Cake.Core" Version="0.28.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
-    <PackageReference Include="RestSharp" Version="106.2.2" />

```

Comment:
Both packages are coming from the [Atlassian.SDK](https://www.nuget.org/packages/Atlassian.SDK) dependencies. I got an inconsistent version error on Newtonsoft.Json if I did a test against inside the test project.